styleguide/_themes/derek/scss/components/rsTabs.scss

```diff
@@ -0,0 +1,126 @@
+.rsTabs {
+  display: block;
+  margin-bottom: 10px;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+}
+
+.rsTabs-menu {
+  list-style: none;
+  margin: 0;
+  margin-left: 15px;
+  padding: 0;
+
+  .rsTabs-tabActive {
+    background-color: $rackspace-red;
+    border: 1px solid $rackspace-red;
+    padding-top: 15px;
+    position: relative;
+    z-index: 5;
+
+    &:hover {
+      background-color: $rackspace-red;
+    }
+
+    .rsTabs-tabLink {
+      color: $white;
+    }
+  }
+}
+
+.rsTabs-tabItem {
+  border: 1px solid $gray-light;
+  border-bottom: 1px solid transparent;
+  display: inline-block;
+  margin-right: -4px;
+  padding: 10px 25px;
+
+  &:not(:first-child) {
+    border-left: 1px solid transparent;
+  }
+
+  &:hover {
+    background-color: $gray-lighter;
+    cursor: pointer;
+  }
+}
+
+.rsTabs-tabLink {
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:focus {
+    text-decoration: none;
+  }
+}
+
+.rsTabs-contentContainer {
+  border-bottom: 2px solid;
+  border-color: $rackspace-red;
+  border-top: 2px solid;
+  min-height: 10px;
+  padding: 10px 0;
+
+  .rsTabs-tabActive {
+    display: block;
+  }
+}
+
+[class^='rsTabs-tabContent-'] {
+  display: none;
+}
+
+
+@media only screen and (max-width: $screen-sm) {
+  .rsTabs-menu {
+    margin-left: 0;
+    text-align: center;
+
+    &::before {
+      color: $white;
+      content: '\f150';
+      font-family: 'FontAwesome';
+      position: absolute;
+      right: 10px;
+      top: 8px;
+      z-index: 10;
+    }
+
+    .rsTabs-tabActive {
+      display: block;
+      padding-top: 8px;
+    }
+  }
+
+  .rsTabs-tabItem {
+    background-color: darken($rackspace-red, 5%);
+    border: 0;
+    border-bottom: 1px solid $rackspace-red;
+    display: none;
+    font-size: 10pt;
+    margin-right: 0;
+    padding: 8px 15px;
+
+    &:hover {
+      background-color: $rackspace-red;
+    }
+  }
+
+  .rsTabs-tabLink {
+    color: $white;
+  }
+
+  .rsTabs-tabMobile {
+    display: block;
+  }
+
+  .rsTabs-contentContainer {
+    border: 1px solid $gray-light;
+    border-bottom: 2px solid $rackspace-red;
+    border-top: 0;
+    padding: 5px;
+  }
```

Comment:
yes this is the mobile version of the content container.. it's nested in a media query